pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1629,6 +1629,17 @@ self: super: {
         --replace "'zoxide_executable', 'zoxide'" "'zoxide_executable', '${zoxide}/bin/zoxide'"
     '';
   };
+  LeaderF = super.LeaderF.overrideAttrs {
+    buildInputs = [ python3 ];
+    # rm */build/ to prevent dependencies on gcc
+    # strip the *.so to keep files small
+    buildPhase = ''
+      patchShebangs .
+      ./install.sh
+      rm autoload/leaderf/fuzzyMatch_C/build/ -r
+      find ./ -type f -name '*.so' -exec strip '{}' \; 2>/dev/null
```

Comment:
maybe we can can let fixup phase handle this with [`stripDebugList`](https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-stripDebugList)?

Reply:
The `./install.sh` script builds the libs in the $src dir, and there is no $out dir to leaderf plugin by default, but the fixup phase is done in $out dir.
So I thought there will be extra works if using fixup phase, like copying all the files to $out dir

Reply:
`buildVimPluginFrom2Nix` copies all the files to `$out`, so you shouldn't need to do anything other than setting `stripDebugList`

and issue with the current approach is that it may only work on linux because of the `.so` extension, if you amend the commit message to be `vimPlugins.LeaderF: ...`, ofborg should try to build it on a bunch of platforms

Reply:
Ok, it seems I take something wrong, there do exist $out dir, I have updated my patch to this.
```nix
    buildPhase = ''
      patchShebangs .
      ./install.sh
      rm autoload/leaderf/fuzzyMatch_C/build/ -r
    '';
    stripDebugList = [ "autoload/leaderf/python" ];
  };
```